src/elements/link/link.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Icon from '../icon'
+import styles from './link.css'
+
+import { classNames } from 'utils'
+
+const Link = React.forwardRef(
+  ({ children, className, external = false, ...restProps }, ref) => (
+    <a
+      ref={ref}
+      className={classNames.use(styles.link).join(className)}
+      {...restProps}
+    >
+      {children}
+      {external && <Icon src="#iconhere" alt="external" />}
```

Comment:
The reason for `external` is not in the Icon, it's fine without, I would rather drop it. The prop is a shortcut for `target="_blank" rel="noopener noreferrer"`

Reply:
alt is required prop. I just put something in there.  So you are saying we should drop required alt prop?

Reply:
I do, we should drop the requirement from the `Icon` but it's slightly different, out of this scope.

In the previous comment, I mean that the `external` prop should add `target="_blank"` but the icon itself is optional.

Reply:
Thanks will add it. It still work in progress so I just put blindly things here.